### ISSUE TITLE
The (x) close is not working when user tries to close the error messages displayed.

### ISSUE THREAD

**Issue by AnkitKhatri55555** [systers/language-translation#259]
**Description**:
The (x) close is not working when user tries to close the error messages displayed.
For the user, the button should be working properly so that the user can close the error messages box.

**Mocks**
![screen shot 1939-12-21 at 5 32 06 pm](https://user-images.githubusercontent.com/36541665/37282729-80ea2580-261b-11e8-9797-907a1e4f2199.png)

**Acceptance criteria:**
The code should be modifed so that the cross can works as it should be.
**Update required**
signup.html
**Expected Time:**
5-10 min

**Comment by Rupinderwadali** [comment]
Yes, we either need to remove the cross button or make it clickable.

**Comment by Rupinderwadali** [comment]
@AnkitKhatri55555 do join the Systers slack channel for languagetranslator as well and pick one issue, solve it and send PR, then work on another. One issue at time.

**Comment by SETIADEEPANSHU** [comment]
@AnkitKhatri55555 Are you working on this , or should I pick it ?

**Comment by AnkitKhatri55555** [comment]
I'm working on it

**Comment by SETIADEEPANSHU** [comment]
@AnkitKhatri55555  Can you please specify which all issues mentors have assigned to you .

**Comment by AnkitKhatri55555** [comment]
@SETIADEEPANSHU  i have been assigned #248 issue.
 there are some others on which i'll start working, once iam finished with that
